machines/settings.ts

```diff
@@ -121,6 +134,18 @@ export const settingsMachine = model.createMachine(
         to: (context) => context.serviceRefs.store,
       }),
 
+      updateDefaults: model.assign({
+        appId: (_context, _event) => {
```

Comment:
can we remove the unused parameters _context, _event ?

Reply:
removed